packages/dnb-eufemia/src/fragments/drawer-list/style/_drawer-list.scss

```
@@ -165,9 +165,22 @@
       position: relative;
       z-index: 1; // only to go over &__triangle
 
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
+      // IE11 support
+      @supports not (display: grid) {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+      }
+
+      @supports (display: grid) {
+        display: grid;
+
+        // Use two columns
+        grid-template-columns: 1fr minmax(2rem, auto);
+
+        // use a hight number in order to center the check icon and suffix with "grid-row: 1 / -1;"
```

```suggestion
        // use a height number in order to center the check icon and suffix with "grid-row: 1 / -1;"
```

`hight` to `height`? Not sure 🤔

---

yeah, it should be "high" 🤦‍♂️ like 10 rows is much for that little cell, but we need a number here in order to let the second column be vertically centered. So 10 seems to me a ok number. Its basically, like if you would this in your data:

```js
const data = [{
    content: ['line 1', 'line 2', 'line 3', ... , 'line 10']
  },
  ...
 ]
```